test/testvm.py

```diff
@@ -23,8 +23,6 @@
 import json
 import libvirt
 import libvirt_qemu
-from   lxml import etree
-from   operator import attrgetter
```

Comment:
don't we need that for this? https://github.com/cockpit-project/cockpit/blob/master/test/testvm.py#L934